### ISSUE TITLE
Apply releases tags consistently

### ISSUE THREAD

**Issue by jamesan** [gtkforphp/cairo#21]
tl;dr --  this is a request to rename existing tags and include any missing versioned commits as new tags. Here's the table of tags with their respective commit hash and commit date:

Tag | Abbrev Hash | Commit Date
---|---|---
0.1.0-alpha | 464c82f | 2009-07-05
0.2.0-beta | cff7f3e | 2010-02-03
0.3.0-beta | c1a3bd0 | 2011-12-28
0.3.1-beta | 2b52581 | 2012-02-15
0.3.2-beta | 04f689e | 2012-04-21

You can also see this applied in my fork: https://github.com/jamesan/cairo/releases.

-------

I'm packaging this software for Arch Linux's user-contributed package repo. Part of the package spec includes a derivation of the upstream version string. When the upstream source is a git repo, the standard derivation expects version strings to be managed using tagged releases.

I forked this repo, intending to make a pull request containing consistently named tags including any missing ones, but pull requests don't support pulling in tags. Tags can only be created by repo contributors, so can someone with write access please apply these few tags?

Thanks!

**Comment by auroraeosrose** [comment]
sure, I'll see what I can do

Part of the issue is this was originally an svn repo and migrated and the tagging stuff did not cooperate :(

**Comment by auroraeosrose** [comment]
Sorry this took so long - done

https://github.com/gtkforphp/cairo/releases

bad tagging from git migration is now fixed - thanks!
